Updater/Updater.php

```diff
@@ -165,16 +165,23 @@ protected function updateAssociatedEntity(ReferenceDataInterface $referenceData,
             }
         } elseif ($classMetadata->isCollectionValuedAssociation($propertyPath)) {
             $associatedEntity = [];
+            if (is_string($value)) {
+                $value = array_filter(explode(',', $value));
```

Comment:
I don't really like it, but during an import, the base Reader has no way to tell if a value is a collection or a plain string.